docs/products/postgresql/concepts/pg-audit-logging.md

```diff
@@ -0,0 +1,192 @@
+
+- Investigate an incident with audit logs as a detailed trail of events leading up to the
+  incidents
+- Analyze the root cause of an incident with audit logs providing data on actions and
+  events that may have led to the incident
+
+System performance optimization
+
+- Monitor and analyze system performance to identify bottlenecks.
+- Analyzing audit logs to assess resource utilization patterns and optimize the system
+  configuration
+
+Data recovery and disaster planning
+
+- Use audit logs for data restoration in case of data loss or system failure
+- Analyze audit logs to improve system resilience and disaster planning strategies by
+  identify potential points of failure
+
+Change management and version control
+
+- Use audit logs to keep a record of changes made to databases, software, and
+  configurations, ensuring a proper version control
+
+## Use cases
+
+The audit logging feature has application in the following industries:
+
+- Finance and banking
+
+  Ensuring compliance with regulatory requirements, tracking financial transactions, and
+  detecting fraudulent activities
+
+- Healthcare
+
+  Maintaining the confidentiality and integrity of patient records as well as complying
+  with privacy regulations
+
+- Government and public sector
+
+  Tracking changes in critical systems, secure sensitive data, and meet legal and
+  regulatory requirements
+
+- Information technology (IT) and software companies
+
+  Monitoring access to the systems, tracking software changes, and identifying potential
+  security breaches
+
+- Retail and e-commerce
+
+  Tracking customer data, transactions, and inventory management to ensure data integrity
+  and prevent unauthorized access
+
+- Manufacturing
+
+  Tracking changes to production processes, monitoring equipment performance, and
+  maintaining data integrity for quality control
+
+- Education
+
+  Protecting sensitive student data, tracking changes to academic records, and monitoring
+  system access for security purposes
+
+## Who is eligible
+
+Aiven for PostgreSQL® audit logging requires the following:
+
+- `[Pro Platform](/docs/platform/concepts/pro-platform)`
+- Aiven for PostgreSQL 11+ Pro Plan
+- `avnadmin` superuser role
+
+## How it works
+
+### Activation with predefined settings
+
+To use the audit logging on your service (database) for collecting logs in Aiven for
+PostgreSQL, you need to
+[enable and configure this feature](/docs/products/postgresql/howto/use-pg-audit-logging)
+using the Aiven [console](https://console.aiven.io), [API](https://api.aiven.io/doc/), or
+[CLI](/docs/tools/cli).
+
+### Configuration options
+
+When enabled on your service, the audit logging can be
+[configured](/docs/products/postgresql/howto/use-pg-audit-logging) to match your use case.
+[Audit logging parameters](https://github.com/pgaudit/pgaudit/tree/6afeae52d8e4569235bf6088e983d95ec26f13b7#readme)
+for fine-tuning the feature are the following:
+
+- `pgaudit.targetDatabases`
```

Comment:
@kathia-barahona Do we still have this option in the latest version of the feature? It seems redundant to me.

Reply:
Hi, we don't. pgaudit settings will be applied globally.